canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.jsx

```diff
@@ -74,27 +78,56 @@ class CheckboxsetControl extends React.Component {
 	}
 
 	render() {
+
+
+		const hidden = this.props.state === STATES.HIDDEN;
+		const disabledTip = this.props.state === STATES.DISABLED;
+
 		let controlValue = this.props.value;
 		if (typeof controlValue === "undefined" || controlValue === null) {
 			controlValue = [];
 		}
 		const checkboxes = [];
 		for (var i = 0; i < this.props.control.values.length; i++) {
+			let tooltip = "";
+			if (this.props.control.description && !this.props.tableControl) {
+				tooltip = (
+					<span >{this.props.control.valueDescs[i]}</span>
+				);
+			}
+			const tooltipIcon = isEmpty(tooltip) ? "" : (
+				<Tooltip
+					id={`tooltip-${this.props.control.name}-${i}`}
+					tip={tooltip}
+					link={this.props.control.description.link ? this.props.control.description.link : null}
```

Comment:
link and tooltipLinkHandler shouldn't be set.

Reply:
removed them.